project/ProjectPlugin.scala

```diff
@@ -158,37 +151,21 @@ object ProjectPlugin extends AutoPlugin {
 
   }
 
-  import autoImport._
-
   override def projectSettings: Seq[Def.Setting[_]] =
-    Seq(
-      resolvers += Resolver.bintrayRepo("beyondthelines", "maven"),
-      orgAfterCISuccessTaskListSetting := List(
-        depUpdateDependencyIssues.asRunnableItem,
-        orgPublishReleaseTask
-          .asRunnableItem(allModules = true, aggregated = false, crossScalaVersions = true),
-        orgUpdateDocFiles.asRunnableItem
-      ),
+    scalaMetaSettings ++ sharedReleaseProcess ++ warnUnusedImport ++ Seq(
+      libraryDependencies ++= commonDeps :+ %("slf4j-nop") % Test,
+      Test / fork := true,
```

Comment:
Using sbt-1.x-recommended unified "slash syntax" instead of "x in y", here and in a few other places.